src/library/searchqueryparser.cpp

```diff
@@ -271,3 +276,43 @@ std::unique_ptr<QueryNode> SearchQueryParser::parseQuery(const QString& query,
 
     return pQuery;
 }
+
+QStringList SearchQueryParser::splitQuery(const QString& query) {
+    QStringList queryWordList = query.split(kSplitRegexp, QString::SkipEmptyParts);
+    return queryWordList;
+}
+
+bool SearchQueryParser::queryIsLessSpecific(const QString& original, const QString& changed) {
+    // separate search query into tokens
+    QStringList oldList = SearchQueryParser::splitQuery(original);
```

Comment:
oldList -> oldWordList

Reply:
hehe, sure. originally it was olist > oldList > oldWordList
now that small function is super clear ;)